backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -308,6 +308,7 @@ public void resume () {
 			for (LifecycleListener listener : listeners) {
 				listener.resume();
 			}
+			resume = true;

```

Comment:
Can we move it outside the synchronized block?

Reply:
Good as done